frontend/src/components/UploadImageCard.scss

```diff
@@ -35,15 +35,23 @@
     color: #828282;
   }
 
+  .dragging {
+    background: #f6f8fb;
+  }
+
   .dragDropImage {
     margin: 0 auto;
     width: 338px;
     height: 218.9px;
-    background: #f6f8fb;
     border: 1px dashed #97bef4;
     border-radius: 12px;
 
-    input {
+    img,
+    label {
+      pointer-events: none;
```

Comment:
This is a very nice CSS trick found [here](https://stackoverflow.com/questions/7110353/html5-dragleave-fired-when-hovering-a-child-element?page=1&tab=scoredesc#tab-top).

The way it works is we tell the browser these nested element of the drop zone are never target of cursor events. So when we are dragging image and moved the cursor to the `img` or the `label` element, the drop zone's `onDragLeave` is never triggered so the background color of the drop zone is still that grayish color.